examples/images_contours_and_fields/streamplot_demo.py

```diff
@@ -0,0 +1,82 @@
+"""
+==========
+Streamplot
+==========
+
+A stream plot, or stream line plot, is used to display 2D vector fields. This
+example shows a few features of the stream plot function:
```

Comment:
Again, when it comes to English, I may be totally wrong, but I think "the streamplot function" may be more correct than the current "the stream plot function", as it is refering to the *`streamplot` function* and not to a stream plot. I guess I was not clear enough: it was actually the meaning of my previous comment ^^. However, for the other utterances of "stream plot"/"stream line," I will let a native English speaker (who may be you!) decide if it is better than "streamplot"/"streamline" or not.